src/features/resources/ResourceCard.tsx

```
@@ -0,0 +1,93 @@
+				<div className="flex items-center text-xs text-white/50">
+					<Calendar size={12} className="mr-1" />
+					{formatDate(new Date(resource._creationTime).toISOString())}
+				</div>
+			</div>
+
+			<p className="text-sm text-white/80 mb-3">{resource.description}</p>
+
+			<div className="flex flex-wrap gap-2 mb-3">
+				{resource.tags.map((tag, index) => (
+					<span
+						key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)